api/client.md

```diff
@@ -0,0 +1,646 @@
+---
+description: >-
+  The Client struct represents the HTTP Client. It has methods like Get, Post, etc, they'll create a new Agent instance and parse request url automatically.
+---
+
+# ⚡ Client
+
+## Start request
+Start a http request with http method and url.
+
+{% code title="Signatures" %}
+```go
+// Client http methods
+func (c *Client) Get(url string) *Agent
+func (c *Client) Head(url string) *Agent
+func (c *Client) Post(url string) *Agent
+func (c *Client) Put(url string) *Agent
+func (c *Client) Patch(url string) *Agent
+func (c *Client) Delete(url string) *Agent
+```
+{% endcode %}
+
+## ✨ Agent
+`Agent` is on top of [fasthttp `HostClient`](https://github.com/valyala/fasthttp/blob/master/client.go#L603) which has lots of convenient helper methods and several `send request` methods.
+
+### Parse
+Parse initializes URI and HostClient.
```

Comment:
Sounds like Parse takes a URI as a param

```suggestion
Parse initializes a HostClient.
```

Reply:
> Sounds like Parse takes a URI as a param

Parse parse the total request, users should set URI via agent.Request().